man/systemd.network.xml

```diff
@@ -1734,6 +1734,14 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>RouteMetric=</varname></term>
+          <listitem>
+            <para>Set the routing metric for routes specified by the
+            DHCP server.</para>
```

Comment:
Shouldn't this say that 1024 will be used by default?

Reply:
OK. Updated.